eradiate/scenes/measure/_distantsensorarray.py

```diff
@@ -0,0 +1,218 @@
+from __future__ import annotations
+
+from typing import Dict, List, Tuple
+
+import attr
+import numpy as np
+import xarray as xr
+
+from ._core import SensorInfo, measure_factory
+from ._distant import DistantMeasure
+from ..illumination import DirectionalIllumination
+from ...attrs import documented, parse_docs
+from ...units import symbol
+from ...units import unit_context_kernel as uck
+
+
+@measure_factory.register(type_id="distantarray")
+@parse_docs
+@attr.s
+class DistantArrayMeasure(DistantMeasure):
```

Comment:
Isn't this class supposed to be an interface definition only? It should not be registered to the factory and should inherit from `ABC` (just to mention explicitly that it is an abstract class).
```suggestion
class DistantArrayMeasure(DistantMeasure, ABC):
```